druid-shell/build.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The Druid Authors.
```

Comment:
I think a more standard way to do this would be to add the "shellapi" feature to winapi-rs, rather than adding a build.rs here.

Reply:
Well... that seems to be a **much** more reasonable idea than what I have done. If was a bit foolish by me to assume that there is no solution for this in the `winapi`-crate.

Before I add `"shellapi"` to the list, do you have an idea if there is a specific meaning to how the features of `winapi` are ordered and formatted?
When I see a block like this in a `Cargo.toml`, I always wonder where to add something new and where to put the line breaks afterwards:
```rust
features = ["d2d1_1", "dwrite", "winbase", "libloaderapi", "errhandlingapi", "winuser",
            "shellscalingapi", "shobjidl", "combaseapi", "synchapi", "dxgi1_3", "dcomp",
            "d3d11", "dwmapi", "wincon", "fileapi", "processenv", "winbase", "handleapi"]
```

Reply:
@MaximilianKoestler while I admire your diligence, I don't think there is a clear ordering. For line breaks, a reasonable guideline is to just keep us at 80 columns. In a perfect world ordering would be alphabetical and fixed by `rustfmt`, but I think you should just add yours at the end (probably after a newline)?

Reply:
@cmyr Thanks. I know that I am the type of dev who usually thinks about these kinds of things for one minute too long 😄. I will just go with the smallest delta.

While I am at it: In this project, do you prefer
 - changes after review as a separate commit,
 - changes after review as a separate commit, but squashed during PR merge,
 -  or just force pushing on the PR branch?

Reply:
We are decidedly anarchic. For a small patch, force pushing is fine; at merge time if commits are clearly written or clearly intentional in their structure I will rebase, and if things are more messy I will squash.